archlinuxcn/llvm-git/PKGBUILD

```diff
@@ -190,8 +191,12 @@ package_clang-git() {
 
     pkgdesc="C language family frontend for LLVM (git version)"
     url="http://clang.llvm.org/"
-    depends=("llvm-git=$pkgver-$pkgrel" 'gcc' "python" 'python2')
-    optdepends=('llvm-libs: for compiling with -flto')
+    depends=("llvm-libs-git=$pkgver-$pkgrel" 'gcc')
+    optdepends=('llvm-libs: for compiling with -flto'
```

Comment:
llvm-libs shouldn't be optdepends anymore.
You can copy the note from the official `clang` for `llvm-git` and `python`. I assume python2 is for the scripts listed below it's probably fine to have it without a note. For `perl` if namcap recommended that it should probably have told you why it's needed and you should add that in the note.